portfolio.css

```diff
@@ -0,0 +1,96 @@
+* {
+	margin: 0;
+	font-size: 20px;
```

Comment:
is this to set `html` font-size for `rem` later? It won't make a difference to the code, and it may not even be best practice, but personally I would set the font-size of the `html` element separately from the rest (all the others will be overridden when you set them to `rem` values anyway)

Reply:
Yeah this was set so I could use rem